qf_lib/common/risk_parity_boxes/risk_parity_boxes.py

```diff
@@ -89,15 +89,17 @@ class RiskParityBoxesFactory:
 
     Parameters
     ----------
-    bbg_data_provider: BloombergDataProvider
-        reference to bloomberg data provider
+    data_provider: AbstractPriceDataProvider
+        reference to a class that implements AbstractPriceDataProvider
```

Comment:
reference to the class providing the pricing data

Reply:
Fixed